R/community.R

```diff
@@ -1088,8 +1088,18 @@ cluster_spinglass <- function(graph, weights = NULL, vertex = NULL, spins = 25,
 #' print(ldc)
 #' plot(ldc, g)
 cluster_leiden <- function(graph, objective_function = c("CPM", "modularity"),
-                           weights = NULL, resolution = 1, beta = 0.01,
-                           initial_membership = NULL, n_iterations = 2, vertex_weights = NULL) {
+                           weights = NULL, resolution = 1,
+                           resolution_parameter = deprecated(), beta = 0.01,
+                           initial_membership = NULL,
+                           n_iterations = 2, vertex_weights = NULL) {
+
+  if (lifecycle::is_present(resolution_parameter)) {
+    lifecycle::deprecate_warn("1.5.0.9006",
```

Comment:
the version number here should be `2.0.0` (I think?) to go with the next release. it's only confusing for those who'll install the dev version.

Reply:
should it be `lifecycle::deprecate_soft()` for this version?

Reply:
> the version number here should be `2.0.0` (I think?) to go with the next release. it's only confusing for those who'll install the dev version.

Why would this be confusing? Happy to say 2.0.0 here, but when we release any pre-2.0 release with this in it, it would be incorrect, and might be more confusing. (I now realise that I might have targeted the dev branch due to breaking changes, but they're now no longer breaking). Or am I misunderstanding something here?